src/components/ToastMessage.tsx

```diff
@@ -0,0 +1,13 @@
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+function ToastMessage() {
+  const notify = () => toast('Error');
+
+  return (
+    <div>
+      <button onClick={notify}>Error</button>
+      <ToastContainer />
+    </div>
+  );
+}
```

Comment:
can you delete this since it's not used?

Reply:
Done!